src/Microsoft.Extensions.Logging.EventSource/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "version": "1.0.0-*",
+  "description": "A logger provider that sends data logged through ILogger interfaces to EventSource/EventListener infrastructure",
+  "packOptions": {
+    "tags": [
+      "EventSource",
+      "logging",
+      "ETW"
+    ],
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/aspnet/logging"
+    }
+  },
+  "buildOptions": {
+      "warningsAsErrors": true,
+      "keyFile": "../../tools/Key.snk",
+      "xmlDoc": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.Logging.Abstractions": "1.0.0-*",
+    "System.Diagnostics.Tracing": "4.1.0-*",
+    "System.Threading": "4.0.11-*",
+    "Newtonsoft.Json": "8.0.3"
+  },
+  "frameworks": {
+    "net46": {
```

Comment:
I just noticed the dependencies here looked messed up as well:

- Move "System.Threading": "4.0.11-*" to the netstandard1.5 dependencies node.
- Remove the "System.Runtime": "4.0.0" from dependencies under net46 and.
- If possible, target netstandard1.3 and remove the net46 framework altogether...